### PR TITLE
fixed: PDF endpoint text item issues

### PR DESCRIPTION
**Pull request by theywa**
#312

### CODE REVIEW

File: public/js/src/module/gui.js [hunk 1]
```diff
@@ -486,6 +488,9 @@ function printOcForm() {
                         }
                         if ( format.queries === 'yes' ) {
                             printified = $dn.trigger( 'printify.enketo' );
+                            textPrints.forEach( ( textPrint ) => {
+                                textPrint.dispatchEvent( events.PrintifyText() );
```
Comment (MartijnR) [inline public/js/src/module/gui.js:492]: the event has now been renamed to Printify()

File: public/js/src/module/gui.js [hunk 2]
```diff
@@ -511,6 +519,9 @@ function printOcForm() {
             return new Promise( function( resolve ) {
                 setTimeout( function() {
                     $dn.trigger( 'deprintify.enketo' );
+                    textPrints.forEach( ( textPrint ) => {
+                        textPrint.dispatchEvent( events.DePrintifyText() );
```
Comment (MartijnR) [inline public/js/src/module/gui.js:523]: the event has now been renamed to DePrintify()

File: public/js/src/module/gui.js [hunk 3]
```diff
@@ -486,6 +488,9 @@ function printOcForm() {
                         }
                         if ( format.queries === 'yes' ) {
                             printified = $dn.trigger( 'printify.enketo' );
```
Comment (MartijnR) [inline public/js/src/module/gui.js:490]: Please also use the new event for the DN questions. This means changing the listener in dn-widget.js as well.

File: public/js/src/module/gui.js [hunk 4]
```diff
@@ -511,6 +519,9 @@ function printOcForm() {
             return new Promise( function( resolve ) {
                 setTimeout( function() {
                     $dn.trigger( 'deprintify.enketo' );
```
Comment (MartijnR) [inline public/js/src/module/gui.js:521]: new event pls

File: public/js/src/module/gui.js [hunk 5]
```diff
@@ -498,6 +503,9 @@ function printOcForm() {
                         }
                         if ( format.queries === 'yes' ) {
                             printified = $dn.trigger( 'printify.enketo' );
```
Comment (MartijnR) [inline public/js/src/module/gui.js:505]: new event pls

File: public/js/src/module/gui.js (hunk 1 above)
Comment (MartijnR) [reply public/js/src/module/gui.js:492]: These events on non-DN questions should always be dispatched (so not only when queries==='yes')

File: public/js/src/module/gui.js (hunk 2 above)
Comment (MartijnR) [reply public/js/src/module/gui.js:523]: The DePrintify() event should always be called (not only when !printified)

File: public/js/src/module/gui.js (hunk 1 above)
Comment (theywa) [reply public/js/src/module/gui.js]: code updated

File: public/js/src/module/gui.js (hunk 2 above)
Comment (theywa) [reply public/js/src/module/gui.js]: code updated

File: public/js/src/module/gui.js (hunk 3 above)
Comment (theywa) [reply public/js/src/module/gui.js]: code updated

File: public/js/src/module/gui.js (hunk 4 above)
Comment (theywa) [reply public/js/src/module/gui.js]: got it

File: public/js/src/module/gui.js (hunk 5 above)
Comment (theywa) [reply public/js/src/module/gui.js]: got it

File: public/js/src/module/gui.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (theywa) [reply public/js/src/module/gui.js]: renamed

**Comment by theywa** [comment]
@MartijnR, I'm really sorry for the late updated, just see the feedback

**Comment by MartijnR** [comment]
Perfect. 👍  Thanks!

**Pull request closed without merging**